cmd/serverless-init/cloudservice/cloudrun.go

```diff
@@ -14,60 +14,68 @@ import (
 )
 
 const (
-	//nolint:revive // TODO(SERV) Fix revive linter
 	revisionNameEnvVar      = "K_REVISION"
 	ServiceNameEnvVar       = "K_SERVICE" // ServiceNameEnvVar is also used in the trace package
 	configurationNameEnvVar = "K_CONFIGURATION"
 	functionTypeEnvVar      = "FUNCTION_SIGNATURE_TYPE"
 	functionTargetEnvVar    = "FUNCTION_TARGET" // exists as a cloudrunfunction env var for all runtimes except Go
 )
 
+const (
+	// SpanNamespace is the namespace for the span
+	cloudRunService  = "gcr."
+	cloudRunFunction = "gcrfx."
```

Comment:
This all looks great, but I feel it might have been simpler to read if you had set these as full constants, like `gcrf.function_target`. Can you help me understand why appending the prefix each time you set a tag is the better approach in this case?

Reply:
I can update the code with constants instead. I went with namespace prefix + tag name instead of a constant namespace tag because of the region and project_id tags. This would require a larger code refactor to check for cloudRunFunctionMode outside of cloudrun file instead of passing in the namespace prefix to the TagMap function in gcp.go